spec/spec_helper.rb

```diff
@@ -85,7 +85,7 @@
   # end
 
   config.before(:each) do
-    EmsRefresh.debug_failures = true
+    EmsRefresh.debug_failures = true if defined?(EmsRefresh) && EmsRefresh.respond_to?(:debug_failures)
```

Comment:
Can we just delete the line, then?

Randomly deciding whether we set it based on whether EmsRefresh happens to have already been loaded seems actively unhelpful.